src/core/lib/security/credentials/oauth2/oauth2_credentials.h

```diff
@@ -32,12 +32,14 @@
   "s&subject_token_type=%s"
 
 // auth_refresh_token parsing.
-typedef struct {
+struct grpc_auth_refresh_token {
+  grpc_auth_refresh_token() : quota_project_id(nullptr) {}
```

Comment:
nit: can remove this line and set `quota_project_id = nullptr` below

Reply:
Done